draft-ietf-tls-subcerts.md

```diff
@@ -53,15 +53,17 @@ informative:
 --- abstract
 
 The organizational separation between the operator of a TLS server and the
-certificate authority that provides it credentials can cause problems, for
-example when it comes to reducing the lifetime of certificates or supporting
-new cryptographic algorithms.  This document describes a mechanism to allow TLS
-server operators to create their own credential delegations without breaking
-compatibility with clients that do not support this specification.
+certificate authority restricts the operator in ways not necessarily envisioned
```

Comment:
Maybe an alternative way to say this is 

  "The organizational separation between the operator of a TLS server and the certificate authority
    can sometimes create limitations.  For example, ....

Reply:
Done.